systers_portal/static/css/style.css

```diff
@@ -117,21 +117,33 @@ body > .container, body > .container-fluid {
   background-color: #fff;
   text-align: center;
   border: 0;
+  color: #575757;
   box-shadow: none;
 }
 
+.jumbotron-custom .index-head {
+  font-family: Cairo;
+  color: #8d2165;
+}
+
 .featurette-divider {
   margin: 4em 0;
 }
 
 .featurette-heading {
   font-weight: 300;
+  font-family: Cairo;
+  color:#F89728;
   line-height: 1;
   letter-spacing: -1px;
   margin-top: 0;
   font-size: 3.5em;
 }
 
+.lead {
```

Comment:
Where is this class used?

Reply:
it's used in the subtext under Systers communities and systers initiatives.

Reply:
please link to issue directly not just use its number. it makes accessing to it much easier

Reply:
@monajalal  done